packages/store/test/StoreHook.t.sol

```diff
@@ -54,30 +58,36 @@ contract StoreHookTest is Test, GasReporter {
           address(echoSubscriber),
           BEFORE_SET_RECORD |
             AFTER_SET_RECORD |
-            BEFORE_SET_FIELD |
-            AFTER_SET_FIELD |
+            BEFORE_SPLICE_STATIC_DATA |
+            AFTER_SPLICE_STATIC_DATA |
+            BEFORE_SPLICE_DYNAMIC_DATA |
+            AFTER_SPLICE_DYNAMIC_DATA |
             BEFORE_DELETE_RECORD |
             AFTER_DELETE_RECORD
```

Comment:
a nice thing we can do now that we are operating with bit flags is add shortcuts, e.g. `BEFORE_ALL` and `AFTER_ALL` like

```solidity
uint8 constant BEFORE_ALL = BEFORE_SET_RECORD | BEFORE_SPLICE_STATIC_DATA | BEFORE_SPLICE_DYNAMIC_DATA | BEFORE_DELETE_RECORD;
```